pkg/controller/scyllacluster/resource.go

```diff
@@ -244,6 +244,8 @@ func StatefulSetForRack(r scyllav1.RackSpec, c *scyllav1.ScyllaCluster, existing
 		},
 		Spec: appsv1.StatefulSetSpec{
 			Replicas: pointer.Ptr(r.Members),
+			// Give some time for kube-proxy and LBs to notice Pod and Endpoints/EndpointSlices being ready.
+			MinReadySeconds: 15,
```

Comment:
5s wasn't enough to reliably pass tests. Reason why it was failing is mentioned in comment. Although since you asked to lower readiness probe interval, I aligned this value accordingly.